src/golang.org/exercises/jewels-and-stones/jewels-and-stones.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+)
+
+func numJewelsInStones(J string, S string) int {
+	jewels := make(map[string]struct{})
+	for i := range J {
+		jewels[string(J[i])] = struct{}{}
+	}
+
+	count := 0
+	for i := range S {
```

Comment:
Nit: Using `i` here is a touch misleading, as `i` typically refers to indices.  However this range iterates over the values and not the indices.